exercises/decimal/README.md

```diff
@@ -0,0 +1,61 @@
+# Decimal
+
+# Decimal
```

Comment:
decimal here twice - this implies that it should be removed from description.md, the generator automatically inserts one

Reply:
Good catch; I'll fix that